modules/ROOT/pages/afm-edit-test.adoc

```diff
@@ -0,0 +1,23 @@
+= Create a New Version of an Existing Test
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You cannot edit a test that exists for a monitor, but you can create a new version of one. In the new version, you can change, remove, or add assertions, headers, endpoints, and reporters.
+
+You can create a new version of a test only if the test has not been customized to include features that the *Create a Monitor from Scratch* page does not support. (It is possible to customize a test by downloading it, editing it manually, and then uploading it.) Tests created manually and then uploaded also are not editable if they contain features that the *Create a Monitor from Scratch* page does not support.
+
+== Procedure
+
+. Open the monitor that includes the test that you want to create a new version of.
+. Click the *Tests* tab.
+. Find the test and click the pencil icon directly beneath it.
+. In the *Modify Existing Monitor* page, make your changes to the test.
+
+== Result
+
+After you click *Update*, a new version of the test is listed in the *Tests* section for the monitor. Previous versions of the test are still listed, so that you can still view the data that is associated with them.
+
+A version number is automatically assigned to the new test. The number is based on the version number for the most recent previously created test. For example, suppose that a monitor contains one test and the version number of the test is 1.0.0. If you create a new version of the test, the number assigned to it is 1.0.1. Next, if you create a new version of either test 1.0.0 or 1.0.1, the new version is assigned the number 1.0.2.
```

Comment:
I think you can omit this paragraph by adding that the version number is "incremented" in the previous.
Unless your audience really needs this clarification; you know best.